network/requestTracker.go

```diff
@@ -196,7 +277,7 @@ func (rt *RequestTracker) Accept() (conn net.Conn, err error) {
 			return
 		}
 
-		trackerRequest := makeTrackerRequest(conn.RemoteAddr().String(), "", "", time.Now())
+		trackerRequest := makeTrackerRequest(conn.RemoteAddr().String(), "", "", time.Now(), conn)
```

Comment:
conn is being passed here, but isn't this nil at this point?
It gets initialized further down:

> 		conn = &requestTrackedConnection{Conn: conn, tracker: rt}

Reply:
Never mind. I missed the initialization above.